providers/redfish/firmware.go

```diff
@@ -413,6 +413,11 @@ func (c *Conn) FirmwareInstallStatus(ctx context.Context, installVersion, compon
 		return state, errors.Wrap(err, "unable to determine device vendor, model attributes")
 	}
 
+	// component is not used, we hack it for tests
```

Comment:
Hardcoding this is not ideal, I think we can make the Task methods more generic, since OpenBMC follows standard Redfish

Reply:
OK, Redfish Task is now the default